src/Database/Redis/ManualCommands.hs

```diff
@@ -1244,6 +1255,21 @@ xclaim
 xclaim stream group consumer minIdleTime opts messageIds = sendRequest $
     xclaimRequest stream group consumer minIdleTime opts messageIds
 
+xAutoClaim
+    :: (RedisCtx m f)
+    => ByteString -- ^ stream
+    -> ByteString -- ^ group
+    -> ByteString -- ^ consumer
+    -> Integer -- ^ min idle time
+    -> ByteString -- ^ start ID
+    -> XAutoClaimOpts -- ^ count
+    -> m (f [StreamsRecord])
```

Comment:
Here the parsing fails and always returns Left Reply. Recheck this and also add the test cases.